aegis_gym/envs/aegis_reacher.py

```diff
@@ -1,51 +1,61 @@
-from typing import Optional
-
 import numpy as np
+import time
 import gymnasium as gym
 from gymnasium import spaces
+from .ros_interface import ROSInterface
```

Comment:
Now let's introduce a mock for the ROSInterface.
As an example, create a new file `ros_interface.py`, create over there a dummy class which returns always a constant values for all ~8 methods which are being used here. 

We can automatize an fallback to a dummy class when ROS is not avaiable:
```suggestion
try:
    from .ros_interface import ROSInterface
except ImportError:
    print("ROS not available, selecting communication mock for testing.")
    from .ros_interface_mock import ROSInterfaceMock as ROSInterface
```